### ISSUE TITLE
I am getting the following error:

### ISSUE THREAD

**Issue by gjwgit** [mlhubber/colorize#6]
_Originally posted by @boulaziz in https://github.com/mlhubber/colorize/issues/4#issuecomment-654389596_

I am getting the following error:
./.local/bin/ml color   colorize --interactive Six_chefs.png
Traceback (most recent call last):
  File "./.local/bin/ml", line 11, in <module>
    sys.exit(main())
  File "/home/khaled/.local/lib/python3.6/site-packages/mlhub/__init__.py", line 197, in main
    args.func(args)
  File "/home/khaled/.local/lib/python3.6/site-packages/mlhub/commands.py", line 1061, in dispatch
    errors = errors.decode("utf-8")
AttributeError: 'NoneType' object has no attribute 'decode'

**Comment by gjwgit** [comment]
Please try the latest version of the colorize package to see if that resolves this issue. If not please provide output of

$ ml --version

$ ml installed

Thanks.

**Comment by boulaziz** [comment]
thank you for your reply, please find below the output of the commands requested:

(base) khaled@khaled-Aspire-V3-571G:~$ ml installed
Found 1 model installed in '/home/khaled/.mlhub/'.

colorize     1.5.3  Demonstrate the concept of photo colorization.          

To configure the package:

  $ ml configure <model>

To view the model's README:

  $ ml readme <model>

To list the commands supported by the model package:

  $ ml commands <model>

(base) khaled@khaled-Aspire-V3-571G:~$ ml --version
mlhub version 3.6.0
(base) khaled@khaled-Aspire-V3-571G:~$

**Comment by boulaziz** [comment]
ml --version
mlhub version 3.6.0

**Comment by gjwgit** [comment]
Thanks. Try an upgrade - it may have fixed the problem (upgrade to mlhub version 3.6.1 and colorize 1.5.6)

$ pip3 install --upgrade mlhub
$ ml configure
$ ml install colorize
...

This is tested on Ubuntu 20.04.

**Comment by boulaziz** [comment]
Thanks. This is done but I get:
$ ml color   colorize --interactive Six_chefs.png
Traceback (most recent call last):
  File "color.py", line 20, in <module>
    import toolz
ModuleNotFoundError: No module named 'toolz

And when trying to install toolz, I get:

$ pip install toolz
Requirement already satisfied: toolz in ./anaconda3/lib/python3.7/site-packages (0.9.0)

**Comment by rlimaservinfo** [comment]
> Thanks. This is done but I get:
> $ ml color colorize --interactive Six_chefs.png
> Traceback (most recent call last):
> File "color.py", line 20, in
> import toolz
> ModuleNotFoundError: No module named 'toolz
> 
> And when trying to install toolz, I get:
> 
> $ pip install toolz
> Requirement already satisfied: toolz in ./anaconda3/lib/python3.7/site-packages (0.9.0)

Same error here
BR

**Comment by gjwgit** [comment]
Does it work if you install the OS package:

`$ wajig install python3-toolz
`

**Comment by gjwgit** [comment]
On a clean Ubuntu 20.04 with ml 3.6.1 and colorize 1.5.8, there are issues installing tensorflow and then keras with `ml configure colorize`.

tensorflow-cpu almost gets there but finishes with 

```
ERROR: Could not install packages due to an EnvironmentError: [Error 13] Permission denied: 'top_level.txt'
```

Manually installing 
```
$ pip3 install --user tensorflow-cpu 
$ ml configure colorize
$ pip3 install keras
$ ml demo colorize
```
succeeds.

**Comment by gjwgit** [comment]
```
$ ml --debug demo colorize
```
Shows the command being executed is:

```
(cd /home/kayon/.mlhub/colorize; export _MLHUB_CMD_CWD='/home/kayon/projects/ubuntu'; export _MLHUB_MODEL_NAME='colorize'; export _MLHUB_PYTHON_EXE="/usr/bin/python3"; export PATH="/home/kayon/.mlhub/colorize/.python/home/kayon/.local/bin:$PATH"; export PYTHONPATH='/home/kayon/.mlhub/colorize/.python/home/kayon/.local/lib/python3.8/site-packages'; /usr/bin/python3 demo.py )
```

Notice that PYTHONPATH is wrong!. Apparently it should be 
```
/home/kayon/.mlhub/colorize/.python/usr/local/lib/python3.8/dist-packages:/home/kayon/.local/lib/python3.8/site-packages
```
and all is good! Looks like an mlhub bug.

**Comment by gjwgit** [comment]
Fixed and tested on Ubuntu 20.04.
